sonar-project.properties

```diff
@@ -0,0 +1,22 @@
+# must be unique in a given SonarQube instance
+sonar.projectKey=aerogearups
+# this is the name and version displayed in the SonarQube UI. Was mandatory prior to SonarQube 6.1.
+sonar.projectName=Aerogear Unified Push Server
+sonar.projectVersion=0.0.1
+
+sonar.sources=common/src/main/java,jaxrs/src/main/java,migrator/src/main/java,model/api/src/main/java,model/jpa/src/main/java,push/model/src/main/java,push/sender/src/main/java,sender/src/main/java
+sonar.tests=common/src/test/java,jaxrs/src/test/java,migrator/src/test/java,model/api/src/test/java,model/jpa/src/test/java,push/model/src/test/java,push/sender/src/test/java,sender/src/test/java
```

Comment:
What are `sender/src/main/java` & `sender/src/test/java`? Should they be `service`?